inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/auth/Authentication.java

```diff
@@ -17,11 +17,20 @@
 
 package org.apache.inlong.manager.common.auth;
 
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import java.util.Locale;
 import java.util.Map;
 
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, visible = true, property = "type")
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = DefaultAuthentication.class, name = DefaultAuthentication.DEFAULT),
```

Comment:
It is recommended to specify its specific type through `@JsonTypeDefine` on the subclass, without adding all subclasses on the parent class - refer to the advice of the open-closed principle.

Reply:
modify JosonUtils